docs/qml/qaoa.md

```diff
@@ -5,22 +5,24 @@ Algorithm (QAOA), first introduced by Farhi *et al.* in 2014 [^1].
 Given an arbitrary graph, the MaxCut problem consists in finding a
 graph [*cut*](https://en.wikipedia.org/wiki/Cut_(graph_theory)) which partitions
 the nodes into two disjoint sets, such that the number of edges in the
-cut is maximized. This is a very common combinatorial optimization problem which is
-computationally hard.
+cut is maximized. This is a very common combinatorial optimization problem known to be computationally hard (NP-hard).
 
-The graph used for this tutorial is a randomly generated using the `networkx` library with
+The graph used for this tutorial is an unweighted graph randomly generated using the `networkx` library with
 a $0.5$ probability of having an edge between two arbitrary nodes.
```

Comment:
That's a good point! I will change the probability to ensure that  $p > \frac{1}{n_{nodes}}$ so that we're guaranteed to have a giant component with high probability for all graph sizes.

Reply:
I think I will just set it to $p=0.8$ so we're above the threshold for every size but not explain why, to not overpopulate the tutorial.

Reply:
Yes that's fine.